### DISCUSSION TITLE
Random sort option and 'on this day'

### DISCUSSION THREAD

**Discussion by theantnest** [photoprism/photoprism#2614] [Ideas]
I just imported 40k images dating back over 40 years (the early ones are scans of old family pics). It would be amazing to be able to sort by random to dig down deep and see pictures I haven't seen in years.

Also an 'on this day' feature would be great, showing older images with the same day/ month date as the current day.

Maybe even some kind of home screen in the future where widgets with filters can be placed by the user somehow?

So when you click the Photoprism Icon, instead of going to search, it goes to the home screen, where the user could decide what that shows? Things like favourites, random, on this day, never viewed, etc. The widgets could even be sidescrolling rows or something?

**Comment by graciousgrey** [comment]
I like the idea of a customisable landing page, a few  related issues:

https://github.com/photoprism/photoprism/issues/577
https://github.com/photoprism/photoprism/issues/153
https://github.com/photoprism/photoprism/issues/720
